pkg/tfvars/baremetal/baremetal.go

```diff
@@ -65,6 +65,13 @@ func TFVars(libvirtURI, bootstrapProvisioningIP, bootstrapOSImage, externalBridg
 		driverInfo["deploy_kernel"] = fmt.Sprintf("http://%s/images/ironic-python-agent.kernel", net.JoinHostPort(bootstrapProvisioningIP, "80"))
 		driverInfo["deploy_ramdisk"] = fmt.Sprintf("http://%s/images/ironic-python-agent.initramfs", net.JoinHostPort(bootstrapProvisioningIP, "80"))
 
+		// explicitly set the boot mode to the default "uefi" in case
+		// it is not set
+		bootMode := "uefi"
+		if host.BootMode == "legacy" {
+			bootMode = "bios"
+		}
```

Comment:
This belongs in pkg/types/baremetal/defaults, and can we use the constants defined in baremetalhost_types.go?

Reply:
Actually, maybe since it's a host default it doesn't belong there.

Reply:
The constants defined in the baremetalhost_types.go are not the values ironic expects. So, I could use the value in the conditional statement, but we need the other literals somewhere. Right now they're going to be defined in ironic.go, and we don't want to import that into the installer. Moving them around somewhere else in the BMO code might be possible, and since we haven't landed the implementation PR that includes them we would have to wait anyway so we can talk about that.

Reply:
If it's too cumbersome to refactor we can just have it here, it's small enough for now.